app-admin/stown/stown-1.2.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+# shellcheck shell=bash disable=2034
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=(python3_{10..13})
+
+inherit distutils-r1 readme.gentoo-r1
+distutils_enable_tests unittest
+
+DESCRIPTION="Manage file system object mapping via symlinks"
+HOMEPAGE="https://www.seichter.de/stown/"
+SRC_URI="https://github.com/rseichter/stown/archive/refs/tags/${PV}.tar.gz -> ${P}.gh.tar.gz"
```

Comment:
We generally want to use the pypi sdist if we can (so `inherit pypi` and drop `SRC_URI`), assuming it has tests. And if it doesn't have tests in there, fix that.

Reply:
Tests are included in the PyPI sdist. Documentation is not. That's why I am using the GitHub release tarball.